ci-operator/step-registry/cucushift/installer/check/control-plane-machinesets/cucushift-installer-check-control-plane-machinesets-commands.sh

```diff
@@ -100,6 +104,11 @@ if ! hasCPMS; then
     exit 0
 fi
 
+if [[ ${CLUSTER_TYPE} == "vsphere" ]] && [[ ${version} == "4.15" ]] && [[ ${FEATURE_SET} != "TechPreviewNoUpgrade" ]]; then
```

Comment:
I think we can merge the condition to the above `hasCPMS` function like:
```
    vsphere*)
       if [[ "${FEATURE_SET}" == "TechPreviewNoUpgrade" ]]; then
          REQUIRED_OCP_VERSION="4.15"
      else
          REQUIRED_OCP_VERSION="4.99999"
       fi
        ;;
```

Reply:
looks good. thanks for advice.

Reply:
We do not need this if block any more, right?